### ISSUE TITLE
Shout on znc dont getting correct time

### ISSUE THREAD

**Issue by Jaglag** [erming/shout#370]
Hi when i fresh connect to a znc the bouncer sends a server-time event  with pre given time when its something written.
Should cant get the correct time and show me everything on the same time i login.

heres something about the server time event:

http://wiki.znc.in/Timestamps

Ty!

**Comment by TheDoctorsLife** [comment]
Still a thing.

**Comment by xPaw** [comment]
Yep, it needs to support `server-time` capability. http://ircv3.net/specs/extensions/server-time-3.2.html

**Comment by astorije** [comment]
Hi all,

Do you confirm this is still a bug?
I am not a ZNC user so I cannot help here. @xPaw, would you like to give a shot at a bugfix? It doesn't seem too difficult to me, but again, I don't know anything about this.

**Comment by xPaw** [comment]
It's not a bug, it's an optional extension. You see more details here: http://ircv3.net/irc/

**Comment by xPaw** [comment]
As far as I can tell, this needs to be directly supported by `slate-irc` and `slate-irc-parser` libraries Shout is using, and neither of these libraries seem to have any activity in last two years.

Supporting IRCv3 extensions would be absolutely good idea though.
Here's a comparison what extensions are supported by other clients and servesrs:

http://ircv3.net/software/clients.html
http://ircv3.net/software/servers.html
